app/interactors/projects/destroy.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Projects
+  class Destroy
+    include Interactor
+    delegate :project, to: :context
+    def call
+      project.destroy
+    end
+  end
+end
```

Comment:
Тут ты написал корректно по работе, но некрасиво, хороший вариант все это аккуратно разделять:
```suggestion
module Projects
  class Destroy
    include Interactor
 
    delegate :project, to: :context
    
    def call
      project.destroy
    end
  end
end
```